lib/appium_lib/ios/mobile_methods.rb

```diff
@@ -9,7 +9,8 @@ class << self
       #   ```
       def extended(_mod)
         Selenium::WebDriver::SearchContext.class_eval do
-          Selenium::WebDriver::SearchContext::FINDERS[:uiautomation] = '-ios uiautomation'
+          Selenium::WebDriver::SearchContext::FINDERS[:predicate] = '-ios predicate string'
```

Comment:
We should switch `Selenium::WebDriver::SearchContext.class_eval` `:uiautomation` or `:predicate` and `:class_name`, I think.